volume_windows.go

```diff
@@ -118,3 +190,42 @@ func invoke(f func(aev *wca.IAudioEndpointVolume) (interface{}, error)) (ret int
 	ret, err = f(aev)
 	return
 }
+
+func invokeMicrophone(f func(aev *wca.IAudioEndpointVolume) (interface{}, error)) (ret interface{}, err error) {
```

Comment:
It seems that the only difference is the arguments of `GetDefaultAudioEndpoint`. Can you create `invokeInternal(isMicrophone bool, f func(aev *wca.IAudioEndpointVolume) (interface{}, error)) (ret interface{}, err error)`?
```go
func invoke(f ...) (interface{}, error) {
  return invokeInternal(false, f)
}
func invokeMicrophone(f ...) (interface{}, error) {
  return invokeInternal(true, f)
}
```